### PR TITLE
Add task notes section to rerunning DAGs

### PR DESCRIPTION
**Pull request by kentdanas**
Highlighting new 2.5 feature

### CODE REVIEW

File: learn/rerunning-dags.md [hunk 1]
```diff
@@ -58,6 +58,16 @@ To clear a full DAG run, go to the Tree View in the Airflow UI and then click **
 
 ![Clear DAG Status](/img/guides/clear_dag_ui.png)
 
+### Add notes to cleared tasks and DAGs
+
+As of Airflow 2.5, you can manually add notes to task instances and DAG runs from the Grid View in the Airflow UI.
```
Comment (jwitz) [inline learn/rerunning-dags.md:63]:
```suggestion
As of Airflow 2.5, you can add notes to task instances and DAG runs from the **Grid View** in the Airflow UI. 
```

File: learn/rerunning-dags.md [hunk 2]
```diff
@@ -58,6 +58,16 @@ To clear a full DAG run, go to the Tree View in the Airflow UI and then click **
 
 ![Clear DAG Status](/img/guides/clear_dag_ui.png)
 
+### Add notes to cleared tasks and DAGs
+
+As of Airflow 2.5, you can manually add notes to task instances and DAG runs from the Grid View in the Airflow UI. 
+
+![Add task note](/img/guides/2_5_task_notes.png)
+
+Task instances and DAG runs with notes are indicated with a grey corner in the Grid View.
```
Comment (jwitz) [inline learn/rerunning-dags.md:67]:
```suggestion
If a task instance or DAG run has a note, its grid box is marked with a grey corner. 
```

File: learn/rerunning-dags.md [hunk 3]
```diff
@@ -58,6 +58,16 @@ To clear a full DAG run, go to the Tree View in the Airflow UI and then click **
...
 
+### Add notes to cleared tasks and DAGs
+
+As of Airflow 2.5, you can manually add notes to task instances and DAG runs from the Grid View in the Airflow UI. 
+
+![Add task note](/img/guides/2_5_task_notes.png)
+
+Task instances and DAG runs with notes are indicated with a grey corner in the Grid View. 
+
+This feature is useful for tracking and maintaining visibility of manual changes made to task instances such as rerunning or changing the task status. We recommend utilizing this feature any time task or DAG statuses are altered.
```
Comment (jwitz) [inline learn/rerunning-dags.md:69]:
```suggestion
This feature is useful for tracking manual changes to task instances, such as reruns or task status changes. Astronomer recommends leaving a note on a task or DAG whenever you manually update it through the Airflow UI.
```

**Review by jwitz — APPROVED** [review]

File: learn/rerunning-dags.md (hunk 1 above)
Comment (paolaperaza) [inline learn/rerunning-dags.md:63]:
```suggestion
As of Airflow 2.5, you can add notes to task instances and DAG runs from the **Grid View** in the Airflow UI. This feature is useful for tracking manual changes to task instances, such as reruns or task status changes. Astronomer recommends leaving a note on a task or DAG whenever you manually update a task instance through the Airflow UI.
```

File: learn/rerunning-dags.md [hunk 4]
```diff
@@ -58,6 +58,16 @@ To clear a full DAG run, go to the Tree View in the Airflow UI and then click **
 
 ![Clear DAG Status](/img/guides/clear_dag_ui.png)
 
+### Add notes to cleared tasks and DAGs
+
+As of Airflow 2.5, you can manually add notes to task instances and DAG runs from the Grid View in the Airflow UI. 
+
+![Add task note](/img/guides/2_5_task_notes.png)
```
Comment (paolaperaza) [inline learn/rerunning-dags.md:65]:
```suggestion

To add a note to a task instance or DAG run:

1.  Go to the **Grid View** of the Airflow UI.
2. Select a task instance or DAG run.
3. Click **Details** > **Task Instance Notes** or **DAG Run notes** > *Add Note**
4. Write a note and click **Save Note**

![Add task note](/img/guides/2_5_task_notes.png)
```

@kentdanas Are these steps correct?

File: learn/rerunning-dags.md (hunk 3 above)
Comment (paolaperaza) [reply learn/rerunning-dags.md:69]: ^Opted for keeping our recommendation up top prior to instructions on how to do it

**Review by paolaperaza — APPROVED** [review]
Coupla notes/a suggestion but this looks great @kentdanas !

File: learn/rerunning-dags.md (hunk 4 above)
Comment (kentdanas) [reply learn/rerunning-dags.md:65]: Yep!

**Pull request merged**